pkgs/by-name/co/construct/package.nix

```
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, boost
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "construct";
+  version = "master";
+
+  src = fetchFromGitHub {
+    owner = "Thomas-de-Bock";
+    repo = "${finalAttrs.pname}";
```

It seems the nixpkgs considers use of `pname` in `src` an "anti-pattern" (see #277994 and #278611 for details):

```suggestion
    repo = "construct";
```